src/app/public/modules/file-attachment/fixtures/file-attachment.component.fixture.html

```diff
@@ -9,7 +9,7 @@
     <sky-file-attachment-label
       *ngIf="showLabel"
     >
-      Field Label
+      Choose file
```

Comment:
The latest version of `axe-core` recommends that labels for button elements be equal to the button's inner text.